### DISCUSSION TITLE
Make the "now playing" menu visible by default again

### DISCUSSION THREAD

**Discussion by Mechanizen** [ppy/osu#22928] [Ideas]
In stable you can clearly see the "now playing" menu (the music control panel on the top right) while in Lazer it is hidden in a button at the top.

When installing Lazer the sound level is set to max and it is kinda anoying to have to search in the menu where to stop the music. And that is only for people who know there is a "now playing" menu, a new player may not instinctively search for it and start browsing the settings menu.

I think it would be more convinient if it was similar to stable with visible "stop/play/resume buttons" to avoid searching for sound options in panic upon opening the game.

**Comment by Axilotl17** [comment]
i think the volume issue is a real problem but i think this isn't the solution.

**Reply by Bruno5430** [reply]
basically #20387

**Comment by Kert** [comment]
I often minimize the game and mute music before that for various reasons, but there is no single click button readily available in main menu to do that like there is in stable
